tests/ci/cdk/cdk/aws_lc_analytics_stack.py

```diff
@@ -3,10 +3,9 @@
 
 from aws_cdk import core, aws_codebuild as codebuild, aws_iam as iam, aws_ec2 as ec2, aws_efs as efs
 
-from util.ecr_util import ecr_arn
 from util.iam_policies import code_build_publish_metrics_in_json
-from util.metadata import AWS_ACCOUNT, AWS_REGION, GITHUB_REPO_OWNER, GITHUB_REPO_NAME
-from util.yml_loader import YmlLoader
```

Comment:
Is `util.yml_loader` still needed? If not, we could delete the file

Reply:
Yes, it's used [here](https://github.com/awslabs/aws-lc/blob/736fce4b11417ea07ae0df06913d7e87a3cd58ca/tests/ci/cdk/cdk/windows_docker_image_build_stack.py#L24).